File: frontend/src/app/models/Anuncio.ts
export class Anuncio {
  id?: number;
  nome: string;
  clienteId: number;
  dataInicio: Date;
  dataTermino: Date;
  investimentoDiario: number;
  qtdDias: number;

  constructor() {
    this.id = 0;
    this.nome  = '';
    this.clienteId = 0;
    this.dataInicio = null;
    this.dataTermino = null;
    this.investimentoDiario = 1.0;
    this.qtdDias = 0;
      }
}

File: backend/models/Anuncio.cs
using System;

namespace backend.models
{
     public class Anuncio
     {
          public Anuncio(int id, double investimentoDiario, int clienteId, DateTime dataInicio, DateTime dataTermino, int quantidadeDias, string nome)
          {
               this.Id = id;
               this.Nome = nome;
               this.ClienteId = clienteId;
               this.DataInicio = dataInicio;
               this.DataTermino = dataTermino;
               this.QuantidadeDias = quantidadeDias;
               this.InvestimentoDiario = investimentoDiario;
          }
          public int Id { get; set; }
          public string Nome { get; set; }
          public int ClienteId { get; set; }
          public DateTime DataInicio { get; set; }
          public DateTime DataTermino { get; set; }
          public int QuantidadeDias { get; set; }
          public double InvestimentoDiario { get; set; }
          public Cliente Cliente { get; set; }

          public Anuncio()
          { }

     }
}